src/App.js

```diff
@@ -1,17 +1,130 @@
-import './App.css';
+/* eslint-disable no-console */
+/* eslint-disable react/jsx-props-no-multi-spaces */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable max-classes-per-file */
```

Comment:
We should avoid disabling trivial linter errors; less to explain in the README.

Reply:
I will try and fix these errors